public_html/wp-content/plugins/wordcamp-coming-soon-page/classes/wordcamp-coming-soon-page.php

```diff
@@ -466,6 +475,31 @@ public function block_new_post_admin_notice() {
 		}
 	}
 
+	/**
+	 * Show a notice if Coming Soon is enabled also in Gutenberg block editor.
+	 */
+	public function maybe_show_block_editor_notice() {
+		$settings = $GLOBALS['WCCSP_Settings']->get_settings();
+
+		if ( 'on' !== $settings['enabled'] ) {
+			return;
+		}
+
+		$message = $this->get_notice_message(); ?>
+
+		<script type="text/javascript">
+		( function( wp ) {
+			wp.data.dispatch( 'core/notices' ).createNotice(
```

Comment:
It looks like this is run on every page, so I'm seeing `Cannot read property 'data' of undefined` in the console on the Dashboard.

Reply:
This is still triggering on posts using the classic editor, so I think you need a "is this the block editor" check too